apps/license-api/infra/license-api.ts

```diff
@@ -0,0 +1,63 @@
+import { ref, service, ServiceBuilder } from '../../../infra/src/dsl/dsl'
+import { Base, Client, Firearm } from '../../../infra/src/dsl/xroad'
+
+export const serviceSetup = (): ServiceBuilder<'license-api'> =>
+  service('license-api')
+    .namespace('license-api')
+    .resources({
+      limits: { cpu: '400m', memory: '512Mi' },
+      requests: { cpu: '200m', memory: '256Mi' },
+    })
+    .env({
+      IDENTITY_SERVER_CLIENT_ID: '@island.is/clients/api',
+      IDENTITY_SERVER_ISSUER_URL: {
+        dev: 'https://identity-server.dev01.devland.is',
+        staging: 'https://identity-server.staging01.devland.is',
+        prod: 'https://innskra.island.is',
+      },
+      SI_PUBLIC_IDENTITY_SERVER_ISSUER_URL: {
+        dev: 'https://identity-server.dev01.devland.is',
+        staging: 'https://identity-server.staging01.devland.is',
+        prod: 'https://innskra.island.is',
```

Comment:
Looks like these two env variables are referencing the same values. Could this be refactored to use a single env variable?

Reply:
ah yeah, this is just some copy paste errors